runtime/westend/Cargo.toml

```diff
@@ -147,6 +148,7 @@ std = [
 	"sp-session/std",
 	"pallet-randomness-collective-flip/std",
 	"runtime-common/std",
+	"frame-try-runtime/std",
```

Comment:
Still not sure how this should work, but okay :P

Reply:
yeah these can be removed, they do nothing afaik.

Reply:
they are needed if you want to compile the crate with std and try-runtime features.
the cli is able to compile it because it activate the frame-try-runtime/std feature through try-runtime-cli